scripts/xtensa-build-zephyr.sh

```diff
@@ -7,7 +7,7 @@ set -e
 
 SOF_TOP=$(cd "$(dirname "$0")" && cd .. && pwd)
 
-SUPPORTED_PLATFORMS=(apl cnl icl tgl-h)
+SUPPORTED_PLATFORMS=(apl cnl icl tgl-h imx)
 # Default value, can (and sometimes must) be overridden with -p
```

Comment:
Shouldn't we add imx8 instead of imx here?

Reply:
yes, should be imx8:
```
-SUPPORTED_PLATFORMS=(apl cnl icl tgl-h)
+SUPPORTED_PLATFORMS=(apl cnl icl tgl-h imx8)
```